### DISCUSSION TITLE
Subtractive genre/tag filtering option

### DISCUSSION THREAD

**Discussion by Shakahron** [gotson/komga#1552] [Ideas]
### Describe your suggested feature

Currently, if you have multiple genres/tags selected, Komga will include a series in the results if just one genre/tag matches. Whereas personally I would prefer Komga to remove results that don't contain all selected conditions.

I feel maybe having the option to toggle between additive and subtractive filtering would be helpful to drill into what you're really looking for, be that in an all encompassing way or with more granular options.

![firefox_i5hJXGMaKX](https://user-images.githubusercontent.com/106402227/170775053-dffda38a-618e-4d0d-8720-c0c0c3b56d14.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by Shakahron** [comment]
I'd just like to add to my suggestion that I think giving options for filter types would probably be needlessly complex in both implementing it and for users. So instead I'd like consideration towards perhaps just changing the tag/genre filter to remove results that don't match your conditions as that's how the majority of filters work I think. Apologies for reposting so soon.

**Comment by Blazeflack** [comment]
I posted my thoughts in a similar discussion: https://github.com/gotson/komga/discussions/1523#discussioncomment-9783991

> I would love to be able to use "negative" filters on my tags for example. In an ideal world it would be possible to easily mix and match "positive" and "negative" filters.
> 
> Some other solutions solve this by allowing 1 click for a positive filtering, and 2 clicks for a negative filtering. A 3rd click clears the filtering for that specific item.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.15.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.15.0))
